robot-db/src/main/java/dao/AuthorDAO.java

```diff
@@ -0,0 +1,14 @@
+package dao;
+
+import domain.Author;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Component;
+
+
+/**
+ * This class is responsible for managing Authors in database
```

Comment:
Cut "this class is" please. It's a class Javadoc, you obviously are NOT talking about packages or methods here.